### ISSUE TITLE
[BUG] LGCMode not working?

### ISSUE THREAD

**Issue by ptdev** [deurk/ktx#179]
**Describe the bug**
Hi,
I'm not sure if this is exactly a bug or something missing when compiling, but I've recently compiled the latest ktx from github (in order to add the XonXonX modes into our existing nquakesv install) but then when using LGCMode the end results are not being displayed as previoulsy.
At the end of an LGCMode game we used to get a message stating our result and whether or not we've improved on our previous score, and that score would then be updated on badplace.eu -> maps -> arena -> povdmm4 (or any other maps that support lgcmode)

**To Reproduce**
1. Go into povdmm4
2. Type: /botcmd enable , /1on1 , /lgcmode , /botcmd fill
3. Ready up and play a 3 minute game with the default skill 10 bot
4. Check console at the end for lgc results

**Expected behavior**
The console at the end of the match should state if you have improved on your previous score or not and update badplace.eu

**Environment(please complete the following information):**
 - Operation System: Linux (Debian testing)
 - Processor Architecture: x86_64
 - MVDSV Version: 0.34-beta
 - KTX Version: 1.40-beta (latest master)

**Additional context**
As I stated initially, I'm not entirely sure if this is an actual bug with ktx or if I just didn't have some required package during install for this functionality to work.
I've followed the compile steps exactly as described in the README.md

**Comment by tcsabina** [comment]
Hi,

Could you check on an official 1.39 ktx server? Or any previous build would do.
For the XonXonX feature's score calculation I had to modify to code at a couple of place. It could very well be that I screwed up something with the LGC mode.

Thanks in advance

**Comment by tcsabina** [comment]
I've just did a quick test on nl.kinsky.io:28502. It is running KTX 1.38...
I have not seen anything printed on the console after the game ended, so I am trying to believe that the feature your are missing is not related to the latest changes of KTX.
I could be mistaken, though...

**Comment by tcsabina** [comment]
Could it be that you need to authenticate your client to make this auto-update of badplace.eu work?
https://github.com/ezQuake/ezquake-source/wiki/Authentication

**Comment by ptdev** [comment]
Hey,

I've just tested our current version (1.39) and it's working as intended. So, at the end of the match we get the "your score was X and you did/didn't improve on your previous score of Y" type message.

On the updated version compiled from the master (1.40-beta) that message doesn't appear. The rest of the LGC related statistics (overshaft/undershaft) do appear tough. It seems like it's just some issue when communicating with badplace, which I guess its what it does to compare the current result with our current top result (?)

Not sure about the kinsky.io servers, but you can test LGC on the london.badplace.eu servers or our servers which are the portuguese ones (clanrot.org:2850*)

At least on the previous version (1.39) lgc would work whether you're authenticated or not. That being said I have tried it both ways.

If there's any more info or something that you need for my part, feel free to ask and I'll do my best to accommodate.

Thanks for taking a look at this. 👍

**Comment by ptdev** [comment]
Ok, i've since got some feedback from discord for some more things to try and eventually reached a solution. 

If I just update ktx the issue is as I've described above, everything seems to work correctly except for submission of lgc reports to badplace (and subsequent message if we improved our score or not)

If I update both ktx and also mvdsv, then everything seems to be working correctly, which I have now done and it solved the issue.

So, I'm closing this as it's most likely not an actual ktx bug. My apologies for any inconvenience or time lost on this.
